.gitignore

```diff
@@ -1,24 +1,33 @@
 # The directory Mix will write compiled artifacts to.
-/_build
+/_build/
 
 # If you run "mix test --cover", coverage assets end up here.
-/cover
+/cover/
 
 # The directory Mix downloads your dependencies sources to.
-/deps
+/deps/
 
-# Where 3rd-party dependencies like ExDoc output generated docs.
-/doc
+# Where third-party dependencies like ExDoc output generated docs.
+/doc/
+
+# Ignore .fetch files in case you like to edit your project deps locally.
+/.fetch
 
 # If the VM crashes, it generates a dump, let's ignore it too.
 erl_crash.dump
 
 # Also ignore archive artifacts (built via "mix archive.build").
 *.ez
 
+# Ignore package tarball (built via "mix hex.build").
+crontab-*.tar
+
+# Temporary files for e.g. tests.
+/tmp
```

Comment:
The library does not use `tmp` as far as I know.

Reply:
> The library does not use `tmp` as far as I know.

Generated by the most recent `mix new` command, no harm keeping it there.
Let me know.

Reply:
Then we should also either add a trailing slash to it (as all the other dirs) or remove it from all dirs.

Reply:
> Then we should also either add a trailing slash to it (as all the other dirs) or remove it from all dirs.

Done.